src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
+$main-color: #060b35;
+$text-color: #616070;
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-top: 32px;
+    margin-inline: 19px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-top: 40px;
+    margin-inline: 16px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+    margin-top: 4px;
+    margin-left: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    box-sizing: border-box;
+    justify-content: space-between;
+
+    margin-top: 16px;
```

Comment:
Again, consider using shorthand for margin properties to make the code more concise. Replace 'margin-top' and 'margin-inline' with 'margin: 16px 16px 0;'.